docs/components/base/wheeled.md

```diff
@@ -101,12 +101,8 @@ Edit and fill in the attributes as applicable.
     },
     {
       "attributes": {
-        "left": [
-          "leftm"
-        ],
-        "right": [
-          "rightm"
-        ],
+        "left": ["leftm"],
+        "right": ["rightm"],
```

Comment:
I know this is JSON, but conceptually this feels like clobbering with the python linter we also just deployed (py linter enforces expansion of some nested components to new lines, for example, counter to what is happening here)

Will prettier ever try to fix python indenting already handled by the py lintiner? Or vice versa? Or are they carefully scoped: (prettier = MD and JSON but skip py code fences || pylintything = py code fences only)?

If so, no worries, should be good!

Reply:
Prettier skipx python code snippets so all good 😊.